aten/src/ATen/native/Linear.cpp

```diff
@@ -212,7 +215,7 @@ Tensor einsum(std::string eqn, TensorList tensors) {
             num_output_dims++;
           }
         }
-      } else {                              // letter (hopefully)
+      } else if (! isspace(c)) {                              // letter (hopefully)
```

Comment:
Didn't read the einsum implementation. Just wondering if this is still necessary after the addition of line 140 above?

Reply:
This is in the right hand side, the above only deals with the left hand side. I could copy the original string and remove the spaces at the top, if that makes the code cleaner.

Reply:
No need. I just wanted to make sure. :) Thanks a lot!